hescorehpxml/__init__.py

```diff
@@ -1101,7 +1101,16 @@ def _get_skylights(self, b):
                     skylight_type_areas[skylight_code] = area
             zone_skylight['skylight_method'] = 'code'
             zone_skylight['skylight_code'] = max(skylight_type_areas.items(), key=lambda x: x[1])[0]
-
+        skylight_sunscreen_areas = {}
+        for skylight in skylights:
+            solar_screen = bool(xpath(skylight, 'h:Treatments/text()') == 'solar screen'
+                                or xpath(skylight, 'h:ExteriorShading/text()') == 'solar screens')
+            area = convert_to_type(float, xpath(skylight, 'h:Area/text()'))
+            try:
+                skylight_sunscreen_areas[str(solar_screen)] += area
+            except KeyError:
+                skylight_sunscreen_areas[str(solar_screen)] = area
+        zone_skylight['solar_screen'] = json.loads(max(skylight_sunscreen_areas.items(), key=lambda x: x[1])[0].lower())
```

Comment:
You could skip the whole `json.loads` thing by not converting the the dict keys above to strings (i.e. `skylight_sunscreen_areas[solar_screen]`)

```suggestion
        zone_skylight['solar_screen'] = max(skylight_sunscreen_areas.items(), key=lambda x: x[1])[0]
```

Reply:
Great suggestion, I tried that at the beginning which led to some bugs I thought it might be allowed to use bool as key, so I made it this way. But I retried today it worked. Thanks!